CHANGELOG.md

```diff
@@ -1,6 +1,95 @@
-## v8.1.2 (Backport of v12.0.1)
+## v13.2.0 (Upcoming)
+
+* Clarify error message when your old and new passwords match, you will need to update translations.
```

Comment:
If translations need to be updated, this is a backwards incompatible release. `v14`, please